src/inputs/select.scss

```diff
@@ -64,3 +66,53 @@
     }
   }
 }
+
+@mixin iui-multi-select-tag-container {
+  position: absolute;
+  inset: 0 $iui-icons-default + $iui-l 0 $iui-sm;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  gap: $iui-xs;
+}
+
+@mixin iui-multi-select-tag {
+  @include iui-tag;
+  margin: 0;
+  display: inline-flex;
+  align-items: center;
+  height: 80%;
+  max-height: $iui-baseline * 3;
+
+  @at-root .iui-small#{&} {
+    .iui-multi-select-tag-label,
+    .iui-multi-select-tag-button {
+      font-size: $iui-font-size-small;
```

Comment:
I feel like we could avoid nesting these classes because `font-size` gets inherited. But I see both of them reset the font-size, so maybe not.

We could still move it into the mixins
![image](https://user-images.githubusercontent.com/9084735/165950425-1e404632-e323-46bd-a48d-4032dc1de642.png)

Reply:
But I guess this selector, which might be messed up, if there are other components with `iui-small`
![image](https://user-images.githubusercontent.com/81580355/166228583-bdf5a030-7866-40e1-9730-83dac7fb3759.png)

Unless I could write something different?

Reply:
What about `@at-root .iui-select.iui-small`? In the future we can change the modifier to be `iui-select-small` maybe.

Reply:
Yeah, that helps!
pushed